src/Qdrant.Client/QdrantClient.cs

```diff
@@ -3306,8 +3306,8 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
 	/// </param>
 	public async Task<IReadOnlyList<ScoredPoint>> DiscoverAsync(
 		string collectionName,
-		TargetVector target,
```

Comment:
may we keep old order of arguments?

Reply:
The target argument accepts a default null now. The context argument doesn't.

So context needs to be first if we do this way

Reply:
Oh, yeah. You have set default value for this arg. Okay, I guess it's fine

Reply:
LGTM, but I recommend to wait remarks from @russcam

Reply:
This is an API breaking change which will break users on version upgrade, and force them to change their code if they are using the discover API.

Is there an agreed versioning scheme (e.g. [semver2.0](https://semver.org/)) that client libraries are following? The change is technically what needs to be done to make `target` optional, but it behooves Qdrant to agree and publicize expectations (in the README and in [client libraries documentation](https://qdrant.tech/documentation/interfaces/#client-libraries)) on version upgrades for consumers, so that they can make informed decisions on when to upgrade for particular version numbers.